### ISSUE TITLE
Multicast packets not being picked up at destination

### ISSUE THREAD

**Issue by verslyap** [alsmith/multicast-relay#15]
I have multicast-relay running on my openwrt router at home.  The output from it shows that SSDP packets are being forwarded by the mutlicast-relay and routed, but if I watch tcpdump on any of my endpoints, almost all of them are dropped.  Maybe every few hours one of them will get through.

Let me know if you need anything to further look at this issue.

**Comment by alsmith** [comment]
Do you have tcpdump capture files taken simultaneously on the following interfaces:

* sending device (or a device on the sending side's network)
* openwrt receiving interface
* openwrt sending interface
* receiving device (or a device on the receiving side's network)

**Comment by verslyap** [comment]
I can get all of them except the sending device (it is a nas).  I'll collect them tonight.

**Comment by verslyap** [comment]
Well, just for the record.  It's openwrt's package of python2 (with the changes outlined in your documentation) but I'm the most recent version of multicast-relay available here on github.

I ran it with the following options:
multicast-relay.py --interfaces br-lan eth0.3 --noMDNS --noSonosDiscovery --homebrewNetifaces --ifNameStructLen=32 --verbose
And the output seemed to confirm that it was working:
(('Adding multicast relay for 239.255.255.250:1900 (SSDP)',), {})
root@router:~# ip mroute
root@router:~# ip mroute show
root@router:~# ip mroute show all
root@router:~# lsmod | grep m(('[SSDP] Relayed 308 bytes from 192.168.12.203 on eth0.3 [ttl 4] to 239.255.255.250:1900 via br-lan/192.168.11.1',), {})
(('[SSDP] Relayed 351 bytes from 192.168.12.203 on eth0.3 [ttl 4] to 239.255.255.250:1900 via br-lan/192.168.11.1',), {})
(('[SSDP] Relayed 355 bytes from 192.168.12.203 on eth0.3 [ttl 4] to 239.255.255.250:1900 via br-lan/192.168.11.1',), {})
(('[SSDP] Relayed 297 bytes from 192.168.12.203 on eth0.3 [ttl 4] to 239.255.255.250:1900 via br-lan/192.168.11.1',), {})

One thing I noticed as I looked at it tonight is that ip mroute show didn't return any routes, even though it was writing out this output.

Tcpdump from the gateway of the source:
https://pastebin.com/DBaQSCCY

Tcpdump matching up the timestamp of the gateway of the destination shows no multicast traffic:
https://pastebin.com/c67R1p8k

Because ssdp packets aren't leaving the router, I omitted the tcpdump from the dlna client.  But none of it showed up in that tcpdump either.
